frontend/src/app/main/ui/ds/controls/select.mdx

```diff
@@ -0,0 +1,66 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import * as SelectStories from "./select.stories";
+
+<Meta title="Controls/Select" />
+
+# Select
+
+Select lets users choose one option from an options menu.
+
+## Variants
+
+**Text**, we will use this variant when there are enough space and icons don't add any useful context.
```

Comment:
typo: `,` -> `:` and start the next sentence with capital case. Same for the sentence below